### DISCUSSION TITLE
Wrong remote address

### DISCUSSION THREAD

**Discussion by Wunderharke** [nextcloud/all-in-one#3949] [Questions] [answered]
I am using nginx on my host and prox to the container on port 11000. X-Forwarded-For should be set correctly however my rexternal ip is recognizes as 172.24.0.1 (docker gateway). Is there anything I have to change inside aio?

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/2045?
